js/lib/serverConnection.js

```diff
@@ -1,10 +1,9 @@
 import { initWebSocket } from './websocket.js';
 import { initSubscription } from './sse.js';
 
-
-export const serverConnection = await (async () => {
-  return await initServerConnection();
-})();
+export async function initServer() {
+  window.warpAO.server = await initServerConnection();
```

Comment:
why are we changing this?
I think there is nothing wrong with top level await
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#top_level_await

Reply:
Ok... it's because we're updating the process id. 
I suggest adding something like switchProcess function next to subscribe and unsubscribe